code/_onclick/click.dm

```diff
@@ -113,9 +113,9 @@
 		var/obj/mecha/M = loc
 		return M.click_action(A,src,params)
 
-	if(restrained())
+	if(HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
 		changeNext_move(CLICK_CD_HANDCUFFED)   //Doing shit in cuffs shall be vey slow
-		RestrainedClickOn(A)
+		UnarmedAttack(A)
 		return
```

Comment:
Will this not allow actions while cuffed, as well as allowing monkeys to attack_paw anything while cuffed?

Reply:
How? 

The trait blocks the use of your hands. It does functionally the same thing as before.

https://discordapp.com/channels/427337870844362753/943311132007997490/1107705086169321502